frontend/src/metabase/nav/containers/AppBar/AppBar.unit.spec.tsx

```diff
@@ -6,7 +6,7 @@ import { setupCollectionsEndpoints } from "__support__/server-mocks";
 import { renderWithProviders } from "__support__/ui";
 import { DEFAULT_EMBED_OPTIONS } from "metabase/redux/embed";
 import { createMockCard } from "metabase-types/api/mocks";
-import type { EmbedOptions } from "metabase-types/store";
+import type { InteractiveEmbeddingOptions } from "metabase-types/store";
```

Comment:
there was a conflict in this file, which is the only file that contains conflict in this backport.